bin/gen-pgbouncer-conf.sh

```diff
@@ -5,23 +5,36 @@ POOL_MODE=${PGBOUNCER_POOL_MODE:-transaction}
 SERVER_RESET_QUERY=${PGBOUNCER_SERVER_RESET_QUERY}
 n=1
 
+# add TLS configuration: for client side of pgbouncer will be "require", for server side of pgbouncer will be "verify-ca". 
+# we need root.crt , certificate and key of pgbouncer on place for verifying, for handling secrets in security compliant manner 
+# they will be encrypted and come from SEO
+
+CLIENT_TLS_KEY_FILE=${CLIENT_TLS_KEY_FILE:-$INTERNAL_TLS_KEY}
```

Comment:
why have a default of `INTERNAL_TLS_*` instead of just `CLIENT_TLS_KEY_FILE=${CLIENT_TLS_KEY_FILE}`?

Reply:
"CLIENT_TLS_KEY_FILE" and "INTERNAL_TLS_KEY" are just names for the same content after the expression.  I could just "CLIENT_TLS_KEY_FILE=${INTERNAL_TLS_KEY}.

Reply:
Unless you plan to have some overriding depending on which variables are provided (or not) I think it might be simpler.